<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RendererConfigWidget</class>
 <widget class="QDockWidget" name="RendererConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Render View</string>
  </property>
  <widget class="QWidget" name="content">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QLabel" name="relatedRenderView">
      <property name="text">
       <string>&lt;b&gt;(No Render View selected)&lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="interactionModeWidget" native="true">
      <layout class="QFormLayout" name="customWidgetsLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&amp;Interaction Mode</string>
         </property>
         <property name="buddy">
          <cstring>interactionModeCombo</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="interactionModeCombo"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QWidget" name="cameraButtonsWidget" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="1">
           <widget class="QPushButton" name="zoomToDataButton">
            <property name="text">
             <string>Zoom To Data</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="resetCameraButton">
            <property name="text">
             <string>Reset Camera</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="propertyguizeug::PropertyBrowser" name="propertyBrowser"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>propertyguizeug::PropertyBrowser</class>
   <extends>QTreeView</extends>
   <header>propertyguizeug/PropertyBrowser.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>interactionModeCombo</tabstop>
  <tabstop>zoomToDataButton</tabstop>
  <tabstop>resetCameraButton</tabstop>
  <tabstop>propertyBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
